src/toolbox.ts

```diff
@@ -441,9 +463,43 @@ export class AnnotationResizeItem extends ToolboxItem {
                 //temporary solution
                 this.cached_size = subtask.annotations.access[annotation_id].line_size
             }
+            this.set_size_cookie(subtask.annotations.access[subtask.annotations.ordering[0]].line_size, subtask)
             return;
         }
-        return;
+        throw Error("Invalid Operation given to loop_through_annotations")
+    }
+
+    private set_size_cookie(cookie_value, subtask) {
+        let d = new Date();
+        d.setTime(d.getTime() + (10000 * 24 * 60 * 60 * 1000));
```

Comment:
what is the utility of this?

Reply:
Sets when the cookie expires. In this example 100 years from when its created

Reply:
I will be filing a bug report in 2122